source-postgres/main.go

```diff
@@ -127,6 +128,11 @@ func (c *Config) Validate() error {
 			}
 		}
 	}
+	if c.Advanced.SSLMode != "" {
+		if !stringMatches(c.Advanced.SSLMode, []string{"disable", "allow", "prefer", "require", "verify-ca", "verify-full"}) {
```

Comment:
I don't see "allow" in the enum list in the config

Reply:
Good catch. I was going back and forth on whether to include that one because it's so amazingly worthless as a client option, and I guess I forgot to update one or the other location. I'll go ahead and include it everywhere for completeness.